pymc/sampling.py

```diff
@@ -1958,13 +1958,14 @@ def sample_posterior_predictive_w(
             # TODO sample_posterior_predictive_w is currently only work for model with
             # one observed.
             # XXX: This needs to be refactored
-            # ppc[var.name].append(draw_values([var], point=param, size=size[idx])[0])
-            raise NotImplementedError()
+            ppcl[var.name].append(draw([var])[0])
```

Comment:
This won't be conditioned on the posterior samples. 

You will have to create a custom function similarly to what `sample_posterior_predictive` does. See recent changes in https://github.com/pymc-devs/pymc/pull/5759

Reply:
It might also be useful to compare what changed between V3 and V4 in `sample_posterior_predictive` because some of the things in `sample_posterior_predictive_w` should no longer be needed (such as the progressbar for instance)